src/io/vtk/vtk-unstructured.h

```diff
@@ -640,481 +87,156 @@ class Cells;
 
 #include <xsd/cxx/xml/dom/parsing-header.hxx>
 
-/**
```

Comment:
Are these comments not relevant anymore? If they are, we should still have them in this new version of the file

Reply:
Comments are now included